_layouts/projects.html

```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+---
+
+<h1> CUGOS Projects </h1>
+<p> {{ page.description }} </p>
+
+<hr>
+
+{% for p in page.projects %}
+  <div class="project">
+    <h1> {{ p.title }} </h1>
+    <h2> {{ p.description}} </h2>
+    <h3> Check it out: <a href={{ p.url}}> {{ p.url }} </a>  </h3>
+    <h3> Contribute: <a href={{ p.github}}> {{ p.github }} </a> </h3>
```

Comment:
Can we quote the `href`s and make the spacing around the curly braces balanced?

Reply:
Updated!